services/skus/datastore.go

```diff
@@ -1435,10 +1435,10 @@ func (pg *Postgres) recordOrderPayment(ctx context.Context, dbi sqlx.ExecerConte
 }
 
 func (pg *Postgres) updateOrderExpiresAt(ctx context.Context, dbi sqlx.ExtContext, orderID uuid.UUID) error {
-	orderTimeBounds, err := pg.orderRepo.GetTimeBounds(ctx, dbi, orderID)
+	expiresAt, err := pg.orderRepo.GetExpiresAtP1M(ctx, dbi, orderID)
```

Comment:
Lets keep it generic `GetTimeBounds` see next comment.

Reply:
Please have a look at #1874. A note on naming: since the method no longer returns bounds, strictly speaking, the old name no longer makes sense. It returns a ready-to-use value for `expires_at`, and the name in the new PR reflects that – `GetExpiresAtAfterISOPeriod`.